src/components/report/report.scss

```diff
@@ -33,3 +33,7 @@
 .row {
   display: flex;
 }
+
+.report-all {
```

Comment:
not sure if this is a css thing but in terms of usability i think the font size is a lot smaller on the report page now?

Reply:
not a huge deal but idk if we want them trying to zoom in during eval